about/page-index/page-index.js

```diff
@@ -7,6 +7,11 @@ module.exports = [
     excerpt:
       "Additional information about Timescale, including how to contribute, and release notes",
     children: [
+      {
+        title: "Changelog",
+        href: "changelog",
+        excerpt: "Timescale Changelog",
```

Comment:
I'd stick with just Changelog. Why? Timescale is now the company, Timescale Cloud is a product, as is PopSQL etc.

Reply:
8b64ba603b5eaa319f4f59ec4431258f40f8f0b2